docs/docs/orgs/README.md

```diff
@@ -1,5 +1,13 @@
 # Managing your Organization
 
+::: info
+
+Improvements are coming. Soon Organizations are transitioning to Workspaces.
```

Comment:
Was there a core reason why we can't just document everything re: orgs + workspaces on this page, changing all references to "workspaces"? Since all new users will get workspaces soon, I'd bet they'll be the majority of people coming to these docs. Then you can simplify by keeping all workspace content here, and you don't need this caveat.